medusa/tv/series.py

```diff
@@ -2060,7 +2083,7 @@ def __unicode__(self):
         to_return += u'anime: {0}\n'.format(self.is_anime)
         return to_return
 
-    def to_json(self, detailed=True):
+    def to_json(self, detailed=True, episodes=False):
```

Comment:
Same as above, was the change to the default response intended?

Reply:
Yes